src/components/PartyAccountItem/PartyAccountItem.stories.tsx

```diff
@@ -11,8 +11,11 @@ const cdnPath = "https://assets.cdn.io.italia.it/logos/organizations/";
 const partyMockImages: Array<PartyAccount> = [
   {
     image: `${cdnPath}1199250158.png`,
-    name: "Comune di Milano",
-    role: "Referente amministrativo",
+    name: `Commissario straordinario per la realizzazione di
```

Comment:
Maybe it's better to add this item at the end of the array than replace the first one